framework/src/postprocessors/FindValueOnLine.C

```diff
@@ -24,7 +24,7 @@ InputParameters validParams<FindValueOnLine>()
   params.addParam<Point>("start_point", "Start point of the sampling line.");
   params.addParam<Point>("end_point", "End point of the sampling line.");
   params.addParam<Real>("target", "Target value to locate.");
-  params.addParam<unsigned int>("depth", 30, "Maximum number of bisections to perform.");
+  params.addParam<unsigned int>("depth", 36, "Maximum number of bisections to perform.");
```

Comment:
I detect sarcasm. 

I was unable to get convergence to 1e-10 with 30 for most test cases (1e-10 is the default tolerance in this object). This new number seems like a better default if we want to keep the current tolerance level.